README.md

```diff
@@ -12,19 +12,19 @@ to us on [Slack](https://dagster.io/slack), or through a GitHub Issue / Discussi
 
 ## Dagster Deep Dives
 
-| Date       | Title                           | Description                                                                                                                            | Code        | Slides        |
-|------------|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|-------------|---------------|
-| 2024.02.13 | Automations                     | Explore the concepts of _automation_ on the Dagster platform through cron-based schedules, sensors, and auto-materialization policies. | [[code]][1] | [[slides]][2] |
-| 2024.02.20 | Resources & Configurations      | Learn how to a build a platform that promotes reusability and extensibility with configurations and resources.                         | [[code]][3] | [[slides]][4] |
-| 2024.03.05 | Thinking in Partitions          | See how pipelines can be parallelized and scale using Dagster partitions.                                                              | [[code]][5] | [[slides]][6] |
-| 2024.03.19 | Integrating Third Party Systems | -                                                                                                                                      | -           | -             |
-| 2024.04.02 | Enable Data Mesh                | -                                                                                                                                      | -           | -             |
+| Date       | Title                      | Description                                                                                                                            | Code        | Slides        |
+|------------|----------------------------|----------------------------------------------------------------------------------------------------------------------------------------|-------------|---------------|
+| 2024.02.13 | Automations                | Explore the concepts of _automation_ on the Dagster platform through cron-based schedules, sensors, and auto-materialization policies. | [[code]][1] | [[slides]][2] |
+| 2024.02.20 | Resources & Configurations | Learn how to a build a platform that promotes reusability and extensibility with configurations and resources.                         | [[code]][3] | [[slides]][4] |
+| 2024.03.05 | Thinking in Partitions     | See how pipelines can be parallelized and scale using Dagster partitions.                                                              | [[code]][5] | [[slides]][6] |
+| 2024.04.02 | Enable Data Mesh           | Learn how the Data Mesh paradigm maps to Dagster and enables organizations.                                                            | [[code]][7] | -             |
```

Comment:
@tacastillo I've added a link to the code, even though it's in a separate repo. I think it'll be fine to throw the slide deck in here too. What do you think?